src/Serializer/AbstractItemNormalizer.php

```diff
@@ -379,18 +384,28 @@ protected function getAllowedAttributes($classOrObject, array $context, $attribu
     /**
      * {@inheritdoc}
      */
-    protected function isAllowedAttribute($classOrObject, $attribute, $format = null, array $context = [])
+    protected function isAllowedAttribute($classOrObject, string $attribute, string $format = null, array $context = [])
     {
         if (!parent::isAllowedAttribute($classOrObject, $attribute, $format, $context)) {
             return false;
         }
 
+        return \is_object($classOrObject) ? $this->canAccessAttribute($classOrObject, $attribute, $context) : true;
```

Comment:
Isn't it dangerous to do so? We could add a security without an object, isn't it?